project/eScavenge/CLI.py

```diff
@@ -533,12 +532,27 @@ def setUp(self):
 
     def test_create_is_gm(self):
         self.assertEqual(None, self.cli.game)
-        self.assertEqual(game_create, self.cli.command("create"), "Failed to Create Game")
+        self.assertEqual(self.cli.command("create"), "Game Created")
 
     def test_create_not_gm(self):
         self.assertEqual(logout, self.cli.command("logout"), "Failed to logout")
         self.assertEqual(permission_denied, self.cli.command("create"), "Only admin can Create a new Game")
 
+    def test_create_in_progress_game(self):
+        self.assertEqual(game_create, self.cli.command("create"), "Failed to Create Game")
+        self.assertEqual(game_started, self.cli.command("start"), "Failed to Start Game")
+        self.assertEqual(self.cli.command("create"), "Game Failed")
```

Comment:
You start out with (const, function, message) then change to (function, const, no message at all). We should be consistent so that the logs are consistent. I prefer the former over the latter.